tests/testscript.py

```diff
@@ -725,6 +726,72 @@ def is_shutoff():
         controllerVM.fail("find /run/libvirt/ch -name *.xml | grep .")
 
 
+    def test_libvirt_events(self):
```

Comment:
nit: `test_libvirt_events` -> `test_libvirt_event_stopped`

in case we will have more tests, we might want to have multiple functions as well; otherwise the function will be hard to read. But we can postpone this to the point where we need more event tests?